apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -234,10 +259,12 @@ export class InstallManager {
             }
           }
 
-          return this._installCommonModules(shrinkwrapIsUpToDate, options.allowShrinkwrapUpdates)
+          return this._installCommonModules(shrinkwrapIsUpToDate, options.allowShrinkwrapUpdates, options.variant)
```

Comment:
> options.variant [](start = 97, length = 16)

It's kind of annoying to have to pass the variant around to everything that is accessing RushConfiguration. Perhaps variant should be an option that is passed in when we load the RushConfiguration. It is probably unlikely that multiple variants would be managed by a single instance of RushConfiguration/invocation of Rush. That would let you simplify a few files where you are sending "options.variant" around.